packages/js/components/src/experimental-tree-control/types.ts

```diff
@@ -39,6 +39,21 @@ type BaseTreeProps = {
 	 * @param value The unselection
 	 */
 	onRemove?( value: Item | Item[] ): void;
+	/**
+	 * It gives a way to determine whether the current rendering
```

Comment:
I think `provides` instead of `gives` is more appropriate here

Reply:
Thanks